superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigModal.tsx

```diff
@@ -619,4 +625,8 @@ function FiltersConfigModal({
   );
 }
 
-export default React.memo(FiltersConfigModal);
+export default React.memo(
+  connect(null, dispatch =>
+    bindActionCreators({ updateCascadeParentIds }, dispatch),
```

Comment:
how about `useDispatch()` instead of this classic connect way since it's already FC.

Reply:
Great idea! Done.